clans/src/main/java/me/mykindos/betterpvp/clans/clans/commands/subcommands/UnclaimSubCommand.java

```diff
@@ -70,6 +74,15 @@ public void execute(Player player, Client client, String... args) {
                         locationClan.getName());
                 return;
             }
+
+            for (ClanMember clanMember : locationClan.getMembers()) {
+                Optional<Client> clientOptional = clientManager.getObject(clanMember.getUuid());
+                if (clientOptional.isPresent()) {
+                    if (clientOptional.get().isAdministrating()) {
+                        UtilMessage.message(player, "Clans", "You may not unclaim territory from this Clan at this time.");
```

Comment:
you need to return here

You also dont need the clientmanager, you can use the gamermanager to get the gamer, then called gamer.getClient()

Reply:
fixed/implemented